Circumference.java

```diff
@@ -1,13 +1,24 @@
 class Circumference extends ConsoleProgram {
 
   /**
-  * Description
-  * @author:
+  * A program that computes the curumference of a circle given a radius from the user 
+  * @author: A. Emmanuel-Singh
   */
   
   public void run() {
     
-    // start coding here
+    // Declare variables
+    double dblRadius;
+    double dblCircumfrence;
+
+    // Get radious 
+    dblRadius = readDouble("Enter the radious: ");
+
+    // Calculate curcumference 
+    dblCircumfrence = 2 * Math.PI * dblRadius;
+
+    // Output Results
+    System.out.println("The circumfrence of a ciccle with radius " + dblRadius + " is " + dblCircumfrence);
```

Comment:
This code still works as the numbers are the same, the sentence in which they are outputted are just different.